contrib/devtools/iwyu/bitcoin.core.imp

```diff
@@ -1,3 +1,11 @@
-# Nothing for now.
 [
+  # Compiler intrinsics
+  { include: [ "<emmintrin.h>", private, "<immintrin.h>", public ] },
```

Comment:
Same here?

Reply:
> they do not fall within the scope of the IWYU repository.

IWYU has mappings for intrinsics though?
 - https://github.com/include-what-you-use/include-what-you-use/blob/master/gcc-8.intrinsics.imp
 - https://github.com/include-what-you-use/include-what-you-use/blob/master/clang-6.intrinsics.imp

Reply:
> Same here?

Please see https://github.com/include-what-you-use/include-what-you-use/issues/1764.